cardano-crypto-class/src/Cardano/Crypto/KES/Mock.hs

```diff
@@ -47,6 +49,9 @@ data MockKES (t :: Nat)
 -- keys. Mock KES is more suitable for a basic testnet, since it doesn't suffer
 -- from the performance implications of shuffling a giant list of keys around
 instance KnownNat t => KESAlgorithm (MockKES t) where
+    -- We only need 8 bytes for the seed, but in order to satisfy
+    -- @instance MLockedPoolSize (SeedSizeKES (MockKES t))@, we pick 32, one
+    -- of the block sizes for which pools exist.
```

Comment:
It's a remnant of a previous iteration of the code, where I attempted to resolve the pool selection at compile time; the `MLockedPoolSize` typeclass would have one instance for each `Nat` for which a suitable pool exists, but the problem with that was that expressing a notion of something like "for all `n :: Nat` between `33` and `64`, inclusively" at the type level in Haskell - `instance (n '> 32, n '<= 64) => MLockedPoolSize n where` is not something we can have, and the only way we could do this would be to explicitly list all 64 valid sizes.

So instead, I begrudgedly resorted to negotiating the pool selection at runtime, demoting the `Nat` via `natVal`, and then hard-casting the pointer we get back to the demanded size-tagged type. This of course isn't typesafe along the entire chain, but it has the advantage that I can actually write it in Haskell without going insane.

Anyway, that comment is stale and needs to go.